jekyll/_cci2/security.md

```diff
@@ -59,9 +59,10 @@ A few different external services and technology integration points touch Circle
 {% endraw %}
 
 ## Audit Logs
-The Audit Log feature is only available for CircleCI installed on your servers or private cloud. 
+CircleCI Server customers can access the audit log feature from UI.
```

Comment:
```suggestion
CircleCI Server customers can access the audit log feature from the UI.
```

Reply:
Sorry I missed a word out the first time! Must have done that before I had any coffee :-)

Reply:
NP, Thank you for your a lot of help 🎉